consul/structs/structs.go

```diff
@@ -380,7 +380,7 @@ func (r *KeyRequest) RequestDatacenter() string {
 type KeyListRequest struct {
 	Datacenter string
 	Prefix     string
-	Seperator  string
+	Separator  string
```

Comment:
Question for @armon or @ryanuber or @slackpad: will this rename break any third-parties reliant upon `Seperator`?  Is our contract with those third-parties such that names can break from release to release?